automation/test.sh

```diff
@@ -97,6 +97,8 @@ elif [[ $TARGET =~ sig-operator ]]; then
 elif [[ $TARGET =~ sig-monitoring ]]; then
     export KUBEVIRT_PROVIDER=${TARGET/-sig-monitoring/}
     export KUBEVIRT_DEPLOY_PROMETHEUS=true
+elif [[ $TARGET =~ ovn ]]; then
+  export KUBEVIRT_DEPLOY_CDI=false
```

Comment:
but now you don't do `export KUBEVIRT_PROVIDER=${TARGET}` isnt it ?
(`KUBEVIRT_PROVIDER` must be set, unless i miss something)

Reply:
I am going to ignore the CDI option, we will see what happend in the future.